packages/linux/package.mk

```diff
@@ -223,7 +217,7 @@ makeinstall_target() {
   if [ "$BOOTLOADER" = "u-boot" ]; then
     mkdir -p $INSTALL/usr/share/bootloader
     for dtb in arch/$TARGET_KERNEL_ARCH/boot/dts/*.dtb; do
```

Comment:
Instead of copying all dtbs, name could be taken from scripts/uboot_helper. In some cases a lot of unrelated dtbs can be built. Is there any usecase when dtb is build but U-Boot is not used? RPi maybe?

Reply:
At least with imx6 only required dtbs were build: https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/imx6/options#L36
Seems similar situation for RPi/RPi2.
This functionality should work the same after rework.

Reply:
It doesn't matter. This way is just fine as it will work for the "general" image. In the update.sh we can control which dtb is updated.